config/v1/tests/apiservers.config.openshift.io/APIServerServingCerts.yaml

```diff
@@ -0,0 +1,638 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "APIServer"
+crdName: apiservers.config.openshift.io
+tests:
+  onUpdate:
+    - name: Should not error with an invalid persisted namedCertificates in spec
+      initialCRDPatches:
+        - op: remove
+          path: /spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/servingCerts/properties/namedCertificates/maxItems
+        - op: remove
+          path: /spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/servingCerts/properties/namedCertificates/items/properties/names/maxItems
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: APIServer
+        spec:
+          additionalCORSAllowedOrigins:
+            - "foo"
+            - "bar"
```

Comment:
Can we remove this from initial so that the updated is actually different from the initial?

Reply:
right, see if I get one green then I'll add several test cases